typescript/packages/reality-capture/src/service/response.ts

```diff
@@ -0,0 +1,37 @@
+import { DetailedErrorResponse } from "./error";
+
+export class Response<T> extends Array<any> {
```

Comment:
why does this class extend from `Array`? Is this object being used in an Array-like manner anywhere? (E.g array destructuring, accessing index, .map(), .filter(), forEach)